File: Examples/CSharp/AsposePDF/Security-Signatures/ChangePassword.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.SecuritySignatures
{
    public class ChangePassword
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_SecuritySignatures();

            //open document
            Document document = new Document(dataDir+ "ChangePassword.pdf", "owner");
            //change password
            document.ChangePasswords("owner", "newuser", "newowner");
            //save updated PDF
            document.Save(dataDir+ "ChangePassword_out.pdf");
            
            
        }
    }
}

File: Examples/VisualBasic/AsposePDF/Security-Signatures/ChangePassword.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.SecuritySignatures
    Public Class ChangePassword
        Public Shared Sub Run()
            ' ExStart:ChangePassword
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_SecuritySignatures()

            ' Open document
            Dim document As New Document(dataDir & Convert.ToString("ChangePassword.pdf"), "owner")
            ' Change password
            document.ChangePasswords("owner", "newuser", "newowner")
            dataDir = dataDir & Convert.ToString("ChangePassword_out_.pdf")
            ' Save updated PDF
            document.Save(dataDir)
            ' ExEnd:ChangePassword  
            Console.WriteLine(Convert.ToString(vbLf & "PDF file password changed successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace